Fusion release tracks (#8961)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes.md

## April 2026

> - **Fix**: Fixed an issue that could cause run details pages to crash for some Fusion-eligible accounts during assisted upgrade flows. Previously, some <Constant name="fusion" /> release-track values were not handled correctly during version comparison. Added support for `fusion-compatible`, `fusion-extended`, and `fusion-fallback`; these options appear in version dropdowns and run filters based on account feature availability.

Reviewer: ```suggestion
- **Fix**: Improved run details pages for Fusion-eligible accounts during assisted upgrade flows. Added support for `fusion-compatible`, `fusion-extended`, and `fusion-fallback`; these options now appear in version dropdowns and run filters based on account feature availability.
```
